lib/ramble/ramble/application.py

```diff
@@ -279,6 +283,10 @@ def set_modifiers(self, modifiers):
         if modifiers:
             self.modifiers = modifiers.copy()
 
+    def set_experiment_log_file(self, experiment_name):
```

Comment:
Here, instead of setting the attribute, you can do something like:

```
def experiment_log_file(self, logs_dir):
    return os.path.join(
        logs_dir
        self.expander.expand_var_name(keywords.experiment_name)) + \
        '.out'
```

Reply:
Used experiment_namespace instead to create separate files for each app.workload.experiment